### PR TITLE
PE-373 - Adding support for Google Analytics tracking per unit.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds support for Google Analytics tracking per unit, since the plain google tracking event doesn't track units when the student navigates in the courseware.

## Note:

This PR needs the updated Google Analytics track code in the main.html file in the theme.

## Reviewers:

- [ ] @andrey-canon 
- [ ] @amalbas

### CODE REVIEW

File: common/lib/xmodule/xmodule/js/src/sequence/display.js [hunk 1]
```diff
@@ -448,6 +451,23 @@
             this.el.find('.nav-item.active .bookmark-icon-sr').text('');
         };
 
+        /**
+         * Sends the vertical+block id to Google Analytics,
+         * in order to track by unit item.
+         */
+        Sequence.prototype.sendPageViewToGA = function(position) {
+            var element = this.link_for(position);
```
Comment (andrey-canon) [inline common/lib/xmodule/xmodule/js/src/sequence/display.js]: What happens if the position is not valid ?

**Review by AuraAlba — APPROVED** [review]

File: common/lib/xmodule/xmodule/js/src/sequence/display.js [hunk 2]
```diff
@@ -448,6 +451,23 @@
...
+            var usageKey = element[0].attributes['data-id'].value;
```
Comment (andrey-canon) [inline common/lib/xmodule/xmodule/js/src/sequence/display.js]: Could you ensure that the element length  is longer than 0 and that the value returned is valid ?

File: common/lib/xmodule/xmodule/js/src/sequence/display.js (hunk 1 above)
Comment (Squirrel18) [reply common/lib/xmodule/xmodule/js/src/sequence/display.js]: Must it fails, and it's not necessary to send GA event. See 415 line. @andrey-canon

File: common/lib/xmodule/xmodule/js/src/sequence/display.js (hunk 2 above)
Comment (Squirrel18) [reply common/lib/xmodule/xmodule/js/src/sequence/display.js]: See 415 line. @andrey-canon

**Comment by Squirrel18** [comment]
Check again please. @andrey-canon

**Review by andrey-canon — APPROVED** [review]

**Review by ericfab179 — APPROVED** [review]

**Pull request merged**